server/routes/api/documents.ts

```diff
@@ -515,13 +519,15 @@ router.post("documents.restore", auth({ member: true }), async (ctx) => {
   // be caught as a 403 on the authorize call below. Otherwise we're checking here
   // that the original collection still exists and advising to pass collectionId
   // if not.
-  if (!collectionId && !collection) {
+  if (!document.isDraftWithoutCollection && !collectionId && !collection) {
     throw ValidationError(
       "Unable to restore to original collection, it may have been deleted"
     );
   }
 
-  authorize(user, "update", collection);
+  if (!document.isDraftWithoutCollection) {
```

Comment:
```suggestion
  if (collection) {
```

Clearer?

Reply:
Makes sense. `collection` being `null` directly implies that the document isn't published either. `isDraftWithoutCollection` seems redundant. Thanks for the spot :pray: